WORKSPACE

```diff
@@ -8,7 +8,10 @@ apollo_repositories()
 http_archive(
     name = "bazel_skylib",
     sha256 = "1dde365491125a3db70731e25658dfdd3bc5dbdfd11b840b3e987ecf043c7ca0",
-    urls = ["https://github.com/bazelbuild/bazel-skylib/releases/download/0.9.0/bazel_skylib-0.9.0.tar.gz"],
+    urls = [
+        "https://apollo-platform-system.cdn.bcebos.com/bazel_dependence%2Fbazel_skylib-0.9.0.tar.gz",
```

Comment:
1) There is already a  `https://apollo-platform-system.bj.bcebos.com/archive/6.0/ ` used for hosting external packages. Please keep a unified URL for that. Say: `https://apollo-platform-system.cdn.bcebos.com/archive/6.0/${package_name}`.  With `6.0` in the URL, we can differentiate dependencies of each release.

2) Don't forget such links in  the `docker/build/installers/` directory and the `third_party/XXX/workspace.bzl` files.

3) Try to avoid specal characters like `%2F`  in link.

4) Does `https://apollo-platform-system.cdn.bcebos.com` fast enough for non-China users ?

Reply:
3 $2F is generated automatically by system。
4 we keep the origin url for forengn users, add the bos url is for chinese user to avoid Bazel BUILD faild

Reply:
> 3 $2F is generated automatically by system。
> 4 we keep the origin url for forengn users, add the bos url is for chinese user to avoid Bazel BUILD faild

3. Please figure out how  to avoid that.
Similar links under `docker/build/installers/` doesn't have this special character.

4. Keep only BOS CDN and the original URL,  remove Bazel mirrored links.

Reply:
ok